docs/content/integrations/dbt/using-dbt-with-dagster/load-dbt-models.mdx

```diff
@@ -15,113 +15,146 @@ description: Dagster can orchestrate dbt alongside other technologies.
-Let's discuss what this example is doing, specifically the <PyObject module="dagster_dbt" object="load_assets_from_dbt_project" /> function. This function loads dbt models into Dagster as assets, creating one Dagster asset for each model.
+---
 
-When invoked, this function:
+## Step 2: Inspect your Dagster project in Dagster's UI
 
-1. Compiles your dbt project,
-2. Parses the metadata provided by dbt, and
-3. Generates a set of software-defined assets reflecting the models in the project. These assets share the same underlying [op](/concepts/ops-jobs-graphs/ops), which will invoke dbt to run the models represented by the loaded assets.
+Now that you have a Dagster project, you can run Dagster's UI to take a look at it.
 
-<Note>
-  <PyObject module="dagster_dbt" object="load_assets_from_dbt_project" /> is one
-  of two ways you can load dbt models into Dagster, which we recommend for small
-  dbt projects. For larger projects, we recommend using
-  <a href="/integrations/dbt/reference#loading-models-using-load_assets_from_dbt_manifest">
-    <code>load_assets_from_dbt_manifest</code> to load models from a dbt{" "}
-    <code>manifest.json</code> file
-  </a>.
-</Note>
+1. Change directories to the Dagster project directory:
 
-Let's take a look at the arguments we've supplied:
+    ```shell
+    cd jaffle_dagster/
+    ```
 
-- `project_dir`, which is the path to the dbt project
-- `key_prefix`, which is a prefix to apply to all models in the dbt project
+2. To start Dagster's UI, run the following:
 
----
+   ```shell
+   DAGSTER_DBT_PARSE_PROJECT_ON_LOAD=1 dagster dev
+   ```
 
-## Step 2: Define a Dagster code location
+   Which will result in output similar to:
 
-Next, you'll define the code location for your Dagster project. A [code location](/concepts/code-locations), created using the <PyObject object="Definitions" /> object, is a collection of definitions in a Dagster project, such as assets, resources, and so on.
+   ```shell
+   Serving dagster-webserver on http://127.0.0.1:3000 in process 70635
+   ```
 
-Assets loaded from dbt require a dbt resource, which is responsible for firing off dbt CLI commands. Using the <PyObject module="dagster_dbt" object="DbtCli" /> resource, we can supply a dbt resource to the dbt project.
+3. In your browser, navigate to <http://127.0.0.1:3000>. The page will display the assets:
 
-Open the `__init__.py` file, located in `/tutorial_template/tutorial_dbt_dagster`, and add the following code:
+   <Image
+   alt="Asset graph in Dagster's UI, containing dbt models loaded as Dagster assets"
+   src="/images/integrations/dbt/using-dbt-with-dagster/load-dbt-models-dbt-asset-graph.png"
+   width={1090}
+   height={894}
+   />
 
-```python
-import os
 
-from dagster_dbt import DbtCli
-from tutorial_dbt_dagster import assets
-from tutorial_dbt_dagster.assets import DBT_PROJECT_PATH
+---
 
-from dagster import Definitions, load_assets_from_modules
+## Step 3: Build your dbt project, aka materialize your assets
```

Comment:
`aka materialize your assets` in the heading strikes me as odd.

Some alternatives:
- `Build your dbt models in Dagster`
- `Run your dbt models in Dagster`
- `Build and materialize dbt models` (ew)
- `Materialize your dbt models as Dagster assets`

Or something along those lines. Ideas? To be clear, my issue is with the `aka` part of the title, not what it's trying to convey.

Reply:
Hah I had a suspicion you weren't going to let me get away with this one. Your first suggestion sounds good.